app/connectors/GuaranteeBalanceConnector.scala

```diff
@@ -27,14 +27,15 @@ import models.values.ErrorType.{InvalidDataErrorType, NotMatchedErrorType}
 import play.api.Logging
 import play.api.http.{HeaderNames, Status}
 import play.api.libs.json.JsResult
+import services.AuditService
 import uk.gov.hmrc.http.HttpReads.Implicits._
 import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpErrorFunctions, HttpReads, HttpResponse}
 
 import java.time.Instant
 import javax.inject.Inject
 import scala.concurrent.{ExecutionContext, Future}
 
-class GuaranteeBalanceConnector @Inject() (http: HttpClient, appConfig: FrontendAppConfig)(implicit
+class GuaranteeBalanceConnector @Inject() (http: HttpClient, appConfig: FrontendAppConfig, auditService: AuditService)(implicit
```

Comment:
You don't need the AuditService in here as we ended up not using it in this class